controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -519,15 +515,12 @@ func (r *StorageConsumerReconciler) reconcileCephClientCephFSNode() error {
 		return err
 	}
 
-	cephClientCephFSNodeDetails := &ocsv1alpha1.CephResourcesSpec{
-		Kind: r.cephClientCephFSNode.Kind,
-		Name: r.cephClientCephFSNode.Name,
-	}
+	var phase string
```

Comment:
To make it more obvious that we are willing to intentionally use an empty string
```suggestion
phase := "" 
```